Sources/Utility/ArgumentParser.swift

```diff
@@ -315,6 +347,18 @@ public final class ArgumentParser: ArgumentParserProtocol {
         return arg
     }
     
+    /// Creates a subparser to this parser. An argument parser can have only
+    /// one subparser, if this method is called again the previous subparser
+    /// will be over-written.
```

Comment:
if we require sub parsers are first positional argument, we can just ditch this and `Subparser` and directly have `add(subparser:)` in `ArgumentParser`, and we can "activate" sub parsers whenever we have atleast one subparser